autogen/main/variables.tf.tmpl

```diff
@@ -559,6 +559,12 @@ variable "workload_config_audit_mode" {
   type        = string
   default     = "DISABLED"
 }
+
+variable "enable_fqdn_network_policy" {
+  type        = bool
+  description = "Enable FQDN Network Policies on the cluster"
+  default     = false
```

Comment:
As this is optional, would `null` be a better default? https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#enable_fqdn_network_policy